application.js

```diff
@@ -0,0 +1,16 @@
+var shapeRegistry = [];
+var test = new Circle(100, 150, 50, 'brown', 'pink', 30);
+shapeRegistry.push(test);
+
+setTimeout(function() {
```

Comment:
@ShaneChesnutt this shows that after the object is created, we can change it. This is just example code obviously.